docs/connector-development/connector-builder-ui/incremental-sync.md

```diff
@@ -0,0 +1,133 @@
+* At the bottom of the stream configuration page, add a new "Request parameter"
+* Set the key to `order-by`
+* Set the value to `oldest`
+
+Setting the start date in the "Testing values" to a week in the past (`2023-04-09T00:00:00Z` at the time of writing) results in the following request:
+```
+curl 'https://content.guardianapis.com/search?order-by=oldest&from-date=2023-04-09T00:00:00Z&to-date=2023-04-15T10:18:08Z'
+```
+
+The last encountered date will be saved as part of the connection - when the next sync is running, it picks up from the last record. Let's assume the last ecountered article looked like this:
+```
+{
+  "id": "business/live/2023/apr/15/uk-bosses-more-optimistic-energy-prices-fall-ai-spending-boom-economics-business-live",
+  "type": "liveblog",
+  "sectionId": "business",
+  "sectionName": "Business",
+  "webPublicationDate": "2023-04-15T07:30:58Z",
+}
+```
+
+Then when a sync is triggered for the same connection the next day, the following request is made:
+```
+curl 'https://content.guardianapis.com/search?order-by=oldest&from-date=2023-04-15T07:30:58Z&to-date=2023-04-16T10:18:08Z'
+```
+
+The `from-date` is set to the cutoff date of articles synced already and the `to-date` is set to the new "now".
+
+## Advanced settings
+
+The description above is sufficient for a lot of APIs. However there are some more subtle configurations which sometimes become relevant. 
+
+### Step
+
+When incremental syncs are enabled, the connector is not fetching all records since the cutoff date at once - instead it's splitting up the time range between the cutoff date and the desired end date into intervals based on the "Step" configuration (by default it's set to 1 month)
+
+For example if the "Step" is set to 10 days (`P10D`) for the Guardian articles stream described above and a longer time range, then the following requests will be performed:
+```
+
+curl 'https://content.guardianapis.com/search?order-by=oldest&from-date=2023-01-01T00:00:00Z&to-date=2023-01-10T00:00:00Z'
+
+curl 'https://content.guardianapis.com/search?order-by=oldest&from-date=2023-01-10T00:00:00Z&to-date=2023-01-20T00:00:00Z'
+
+curl 'https://content.guardianapis.com/search?order-by=oldest&from-date=2023-01-20T00:00:00Z&to-date=2023-01-30T00:00:00Z'
+
+...
+```
+
+After an interval is processed, the cursor value of the last record will be saved as part of the connection as the new cutoff date.
+
+In most cases, the default step size is fine, there are two reasons to change it:
+* **The API is unreliable** and the cutoff date should be saved more often to prevent resync of a lot of records - if the "Step" size is a day, then at most one day worth of data needs to be resync in case the sync fails halfway through. However, a smaller step size might cause more requests to the API and more load on the system.
```

Comment:
Can we focus on the checkpointing ability instead of blaming the API for being unreliable? This is also useful when Airbyte is responsible for the sync failure